assets/src/edit-story/app/media/utils/useProcessVideo.js

```diff
@@ -108,6 +139,17 @@ function useProcessVideo({
         }
       };
 
+      const onUploadProgress = ({ resource }) => {
+        const oldResourceWithId = { ...resource, id: oldResource.id };
+        updateElementDimensions({
+          id: oldResource.id,
+          resource: oldResourceWithId,
+        });
```

Comment:
Update the dimisons of the element if needed. See useUploadwithPreview, where I got this from.